Makefile

```
@@ -0,0 +1,104 @@
+
+SHELL := /bin/bash
+PYTHON ?= 3.12
+
+.ONESHELL:
+
+.PHONY: help
+help:  # Print this message.
+	@echo 'Usage: make PACKAGE=<package> [PYTHON=<python version>] target [ARGS=<additional args>]'
```

I much prefer specifying the target first and args after (like a regular function call). This works fine.
```suggestion
	@echo 'Usage: make target PACKAGE=<package> [PYTHON=<python version>] [ARGS=<additional args>]'
```

---

Having played with this a little, uppercase `PACKAGE=...` is a bit annoying to type. Maybe we should do `package=...` lowercase for the required args, but keep `PTYHON` and `ARGS` for the optional env vars? But maybe consistency is better than ease.

---

Interestingly you can also set `PACKAGE` as an environment variable, which could be a convenient way to manage it?

I think consistency is worthwhile here. What do you think about lowercase for all of `package`, `python` and `args`? I guess that does potentially cause problems with `python` -- not immediately for us, but it's a footgun waiting to happen.

Alternative idea: `PKG` and `PY` instead of `PACKAGE` and `PYTHON`?

---

Ah yes, setting the package as an env var when you're working on a single package is a good idea.

I like `PKG` and `PY` -- they're shorter, and easier to type with left-shift held down. :-)

---

Done